libraries/esp8266/examples/BlinkPolledTimeout/BlinkPolledTimeout.ino

```diff
@@ -39,7 +39,7 @@ void ledToggle() {
 
 
 
-esp8266::polledTimeout::periodic halfPeriod(500); //use fully qualified type and avoid importing all ::esp8266 namespace to the global namespace
+esp8266::polledTimeout::periodicCpu halfPeriod(500); //use fully qualified type and avoid importing all ::esp8266 namespace to the global namespace
```

Comment:
Did you change this just to have a cpu type in an example (CI coverage), or was there some other reason?